src/main/resources/types/Stats.d.ts

```diff
@@ -1,31 +1,40 @@
 export interface Stats {
-	outputPath: string
+	// outputPath: string
```

Comment:
Dead comments all over the file?

Reply:
I cleaned most, but added two comments, since rspack will remove those. Not that it matters too much since I don't use them for anything.